tiers.js

```diff
@@ -256,8 +273,28 @@ window.addEventListener('dragend', end_drag);
 function make_accept_drop(elem) {
 	elem.classList.add('droppable');
 
+  let target_item_index;
+  let drag_enter_img;
+
 	elem.addEventListener('dragenter', (evt) => {
 		evt.target.classList.add('drag-entered');
+
+		// Grabs the index of the item that the dragged item is hovering.
+		drag_enter_img = evt.target;
+
+		let rows = Array.from(tierlist_div.querySelectorAll(".row"));
+		let parent_div = drag_enter_img.parentNode.parentNode.parentNode;
+		let idx = rows.indexOf(parent_div);
+	
+		if (rows[idx] != undefined) {
+			let image_node_list_count = rows[idx].querySelectorAll("img").length;
+			let image_node_list = rows[idx].querySelectorAll("img");
+			for (let i = 0; i < image_node_list_count; i++) {
+			if (image_node_list[i] == drag_enter_img) {
+				target_item_index = i;
+			}
+			}
+		}
```

Comment:
This whole piece of code is the same as the one in `create_img_with_src`: I'd extract it into a function that returns the item index